### PR TITLE
Update Charter based on SIG meeting feedback

### PR DESCRIPTION
**Pull request by lmbr-pip**
* Update charter based on meeting notes: https://github.com/o3de/sig-network/blob/main/meetings/notes/sig-meeting-20211005.md

Signed-off-by: Pip Potter <rppotter@amazon.com>

### CODE REVIEW

File: governance/SIG Network Charter.md [hunk 1]
```diff
@@ -1,97 +1,99 @@
...
 
 ## In scope
 - Define standard for communication negotiation.
-- Define and maintain implementation to enable compression and secure communication between endpoints
-- Define and implement interface to network counters, monitoring, and statistics
-- Define and implement low level instrumentation APIs
+- Define and maintain implementation to enable compression and secure communication between endpoints.
+- Define and implement interface to network counters, monitoring, and statistics.
+- Define and implement low level instrumentation APIs.
+- Maintenance of the Multiplayer Sample project.
```
Comment (AMZN-Gene) [inline governance/SIG Network Charter.md:29]: Could we hyperlink "[Multiplayer Sample](https://github.com/o3de/o3de-multiplayersample)" -> https://github.com/o3de/o3de-multiplayersample?

**Review by AMZN-Gene — APPROVED** [review]

File: governance/SIG Network Charter.md (hunk 1 above)
Comment (lmbr-pip) [reply governance/SIG Network Charter.md]: Done

File: governance/SIG Network Charter.md [hunk 2]
```diff
@@ -1,97 +1,100 @@
...
 ## Scope
 
 - Generalized overall scope of work
 
 ## In scope
 - Define standard for communication negotiation.
-- Define and maintain implementation to enable compression and secure communication between endpoints
-- Define and implement interface to network counters, monitoring, and statistics
-- Define and implement low level instrumentation APIs
+- Define and maintain implementation to enable compression and secure communication between endpoints.
```
Comment (junbo75) [inline governance/SIG Network Charter.md:26]: Nit: May worth adding could services integration to this general overview section as well.

File: governance/SIG Network Charter.md [hunk 3]
```diff
@@ -1,97 +1,100 @@
...
 - Define and maintain implementation of communication and data transfer between network based tools.
-
-- Reponsible for design and implementation of controller component until matured and moved into simulation SIG.
-
+- Responsible for design and implementation of controller component until matured and moved into simulation SIG.
 - Publish multiplayer component standard architecture to be shared with SIGs for remote simulation (physics/animation)
-
-- Items that span or require other SIGs or groups and how it relates to this SIG’s responsibilities
+- Define and maintain network layer encryption, as a cross-cutting issue with `#sig-security`.
+- Define and maintain network testing, as a cross-cutting issue with `#sig-testing`.
```
Comment (junbo75) [inline governance/SIG Network Charter.md:54]: Does "network testing" contain testing for the cloud service integration as well?

Comment (lmbr-pip) [reply governance/SIG Network Charter.md:54]: In as much as integrations that owned or related to things owned by the SIG.

File: governance/SIG Network Charter.md (hunk 2 above)
Comment (lmbr-pip) [reply governance/SIG Network Charter.md:26]: I think we agreed the SIG is not responsible for specific cloud services integrations unless SIG agrees to accept to them. Can add note is that SIG is responsible for mechanisms that enable cloud service integrations, but not integrations with specific cloud services.

Comment (lmbr-pip) [reply governance/SIG Network Charter.md:26]: This is already covered by "Define and maintain code to enable cloud data services communication."

File: governance/SIG Network Charter.md (hunk 3 above)
Comment (lmbr-pip) [reply governance/SIG Network Charter.md:54]: Added quick note.

**Comment by lmbr-pip** [comment]
PR migrated to https://github.com/o3de/sig-network/pull/34 (due to DCO issue)

**Pull request closed without merging**